models/issue.go

```diff
@@ -1209,7 +1209,10 @@ func GetParticipantsByIssueID(issueID int64) ([]*User, error) {
 	userIDs := make([]int64, 0, 5)
 	if err := x.Table("comment").Cols("poster_id").
 		Where("issue_id = ?", issueID).
```

Comment:
shouldn't this really be?
```sql
Where("`comment`.issue_id = ?", issue).
```

Reply:
Since `issue_id` is a unique attribute in this join it is valid sql, but I can add it for the sake of clarity.